packages/ra-ui-materialui/package-lock.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
please don't commit the packages.json

Reply:
In my memory, this project required to commit it. Am I wrong or did it change?